gensim/test/test_varembed_wrapper.py

```diff
@@ -26,8 +26,8 @@
 # needed because sample data files are located in the same folder
 module_path = os.path.dirname(__file__)
 datapath = lambda fname: os.path.join(module_path, 'test_data', fname)
-varembed_model_vector_file = datapath('varembed_leecorpus_vectors.pkl')
-varembed_model_morfessor_file = datapath('varembed_leecorpus_morfessor.bin')
+varembed_model_vector_file = datapath('varembed_vectors.pkl')
```

Comment:
Yes, I've changed the corpus, the varembed test model was trained on, and hence also the model pickle and morfessor binary.